server/hedley/themes/custom/hedley_theme/css/custom.css

```diff
@@ -1,3 +1,4 @@
-/*=======================================================================
-*** PORTO_SUB STYLE SHEET***
-========================================================================*/
+/* line 1, ../sass/custom.scss */
+.content img {
```

Comment:
@nader77 do we want to introduce SASS here?

I think for now we can go for now with just a simple `css`, and if the need raises introduce sass. Anyway, I'm not sure -- so I'm letting you decide if merge or change PR to include only the css file.